gulpfile.js

```diff
@@ -1,5 +1,4 @@
-/* global require */
```

Comment:
Did you mean to remove this line? (I ask only because I'm not sure what it does/relates to and am being paranoid.)